### PR TITLE
pango version to the last 1.44.0

### PR DESCRIPTION
**Pull request by lillolollo**
update pango version to the last 1.44

### CODE REVIEW

File: mingw-w64-pango/PKGBUILD [hunk 1]
```diff
@@ -4,8 +4,8 @@
 _realname=pango
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=1.43.0
-pkgrel=2
+pkgver=1.44
```
Comment (lazka) [inline mingw-w64-pango/PKGBUILD]: 1.44.0

File: mingw-w64-pango/PKGBUILD [hunk 2]
```diff
@@ -28,7 +28,7 @@ options=('staticlibs' 'strip' 'emptydirs')
 source=("https://download.gnome.org/sources/pango/${pkgver:0:4}/${_realname}-${pkgver}.tar.xz"
         static-skip-help2man.patch
         0001-pango.pc-Make-gobject-2.0-a-non-private-requirement.patch)
```
Comment (lazka) [inline mingw-w64-pango/PKGBUILD]: this patch needs to be dropped

**Comment by lazka** [comment]
meson options changed:

* gir -> introspection
* enable_docs -> gtk_doc

**Comment by lillolollo** [comment]
@lazka Feel free to open a new patch I can close this request since I'm not very skilled  ;-)

**Comment by lazka** [comment]
ok, I'll try locally and do some basic testing first

**Comment by lazka** [comment]
pango 1.44 has some weird win32 font rendering issues, so we should try to backport the windows fixes first

**Pull request closed without merging**